### DISCUSSION TITLE
Subpage links do not direct users to the subpage, rather to the space landing page

### DISCUSSION THREAD

**Discussion by sierratull** [GitbookIO/community#91] [0. General bugs and questions] [answered]
### What's your feature/idea?

Links that are copied from subpages within a space and bookmarked outside of GitBook, takes the user to the landing page of the Space. 

### What's your desired outcome?

Have the ability to use a copied link from a subpage externally (outside of GitBook) and link directly to the subpage and not to the space's landing page

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment, marked as answer]
Hey @sierratull, 

Could you email us at [support@gitbook.com](mailto:support@gitbook.com)? Please include in the email the URL of the space you are working with and pages where you see this happening. 

From there, we'll be in a position to investigate this with you!

Thank you!

> Disclaimer: Please note that this is not our official support channel. If you encounter any bugs or issues, we strongly encourage you to contact [support@gitbook.com](mailto:support@gitbook.com).
